src/test/scala/org/ergoplatform/utils/generators/ValidBlocksGenerators.scala

```diff
@@ -178,7 +178,8 @@ trait ValidBlocksGenerators
     val (adProofBytes, updStateDigest) = utxoState.proofsForTransactions(transactions).get
 
     val time = timeOpt.orElse(parentOpt.map(_.timestamp + 1)).getOrElse(timeProvider.time())
-    val extension: ExtensionCandidate = LaunchParameters.toExtensionCandidate()
+    val interlinks = Seq.empty // todo: use real interlinks
```

Comment:
Looks like we never check that interlinks written in Extension are correct. This check should be done in `FullBlockSectionProcessor`.

Reply:
@catena2w, fixed